api/src/filtersQueryPipe.ts

```diff
@@ -0,0 +1,26 @@
+import { ArgumentMetadata, Query, ValidationPipe } from '@nestjs/common';
+import { IsIn, IsInt, Min, IsString, MaxLength } from 'class-validator';
+
+export class NickQueryDto {
+    @IsString()
+    @MaxLength(20)
+    equal = '';
+}
+
+export class FiltersQueryPipe extends ValidationPipe {
```

Comment:
Why we need `ValidationPipe`? Why we need to validate the query or the nick?

Reply:
https://docs.nestjs.com/pipes#built-in-pipes

Reply:
![image](https://user-images.githubusercontent.com/5002775/211888380-48fa5c5e-ed21-45b9-a01f-5cf382c9e485.png)